tox.ini

```diff
@@ -143,9 +143,7 @@ commands_pre =
 
   test-utils: pip install -r {toxinidir}/tests/opentelemetry-test-utils/test-requirements.txt
 
-  opentelemetry-proto-0: pip install -r {toxinidir}/opentelemetry-proto/test-requirements-0.txt
-  opentelemetry-proto-1: pip install -r {toxinidir}/opentelemetry-proto/test-requirements-1.txt
-  lint-opentelemetry-proto: pip install -r {toxinidir}/opentelemetry-proto/test-requirements-1.txt
+  opentelemetry-proto-protobuf5: pip install -r {toxinidir}/opentelemetry-proto/test-requirements.txt
```

Comment:
Lint is failing prob related to this https://github.com/protocolbuffers/protobuf/issues/10372

Reply:
Any idea how to move forward?

Reply:
@aabmass I think we have  some options:

1. Fix lint separately in another PR (https://github.com/open-telemetry/opentelemetry-python/pull/4209)
2. Assign all top-level globals explicitly in the generate code
3. Skip the lint 😓

Reply:
Merged